### PR TITLE
Revised benchmarks and made use of QUIT_GAP

### PR DESCRIPTION
**Pull request by alex-konovalov**
I have revised five benchmarks that are part of the weekly run test and made use of `QUIT_GAP` there to improve automated failure detection. 

This PR does not change anything that may be tested by automated tests for PRs, and it can be rebased on the master branch.

### CODE REVIEW

**Comment by alex-konovalov** [comment]
The next update of this PR contains new benchmarks for automorphism groups and group isomorphism. They are based on @hulpke's initial (full) version of `tst/teststandard/grpauto.tst` from #896 before it was reduced to avoid running out of time/memory. I am testing the benchmarks at the moment - please do not merge yet.

File: benchmark/grpauto/hardest.tst [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+false
+
+#
+gap> G:=PcGroupCode(18738408935379049727906755356708168311565445686261463850856,
+> 21952);;
+gap> H:=PcGroupCode(18738408935359210460657231881739776911013615108923450662760,
+> 21952);;
+gap> IsomorphismGroups(G,H);
+fail
+gap> IsomorphismGroups(G,PcGroupCode(CodePcGroup(G),Size(G)))=fail;
```
Comment (alex-konovalov) [inline benchmark/grpauto/hardest.tst]: Still in line 66 and using now almost 30g (and unlike in files from `tst`, this uses default assertion level. I wonder whether that test was ever runnable at all. Reminder to myself: next time watch more carefully for PRs with tst files which are not covered by the Travis entry check.

**Comment by hulpke** [comment]
@alex-konovalov 
I just tried this last line in my `work` branch (which should be minimally different from the `additions` branch you merged, and this last example ran through in 50 seconds.

However random choices (e.g. for permutation representations) can come into play, and there is a number of heuristics that decide whether (e.g.) some permutation representation is ``good enough''. This can be play out differently depending on random choices and yes, I only ran individual tests, not the whole file.

I will look at what happens in sequence with the previous tests.

File: benchmark/grpauto/hardest.tst (hunk 1 above)
Comment (fingolfin) [inline benchmark/grpauto/hardest.tst]: For the record, on my system the line 64 check takes 10 minutes, the line 66 one 30 seconds.

**Comment by alex-konovalov** [comment]
I have rebased this onto the current master and now the problematic `benchmark/grpauto/hardest.tst` runs as well. So I think is is now ready for merge. Three commits are logically independent, so it should be fine to rebase this onto master. I am running these benchmarks weekly with Travis, and this PR will be helpful since now they should properly fail the test if they do not pass.

**Pull request merged**